src/model/Keyword.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { Fragment } from 'react';
+import { v1 } from 'uuid';
+import { InfoList } from 'franklin-sites';
+import apiUrls from '../utils/apiUrls';
+import EntrySectionType from './types/EntrySection';
+import {
+  entrySectionToKeywordCategories,
+  KeywordCategory,
+} from './types/keywordTypes';
+
+type Keyword = {
+  id?: string;
+  value?: string;
+  category?: KeywordCategory;
+};
+
+type Data = {
+  keywords?: Keyword[];
+};
+
+type KeywordProps = {
+  section: EntrySectionType;
+  data: Data;
+};
+
+type KeywordListProps = {
+  keywords?: Keyword[];
+};
+
+type KeywordItempProps = {
+  id?: string;
+  value?: string;
+};
+
+export const KeywordItem: React.FC<KeywordItempProps> = ({ id, value }) => {
+  if (!id || !value) {
+    return null;
+  }
+  return <a href={apiUrls.keyword(id)}>{value}</a>;
```

Comment:
This will eventually be a `Link` pointing to `/keywords/KW-0418` rather than an anchor. I know the url won't be working but I wonder if it would be better to link to "what will be" so we don't have to change these links again later?

Reply:
Updated to use `Link` point to `/keywords/KW-XXXX`.